scripts/downloader.py

```diff
@@ -0,0 +1,77 @@
+import asyncio
+import os
+
+from requests import get
+
+
+DOWNLOAD_DIR = './_downloaded'
+PROJECT_URL = 'https://circleci.com/api/v1.1/project/github/honzajavorek/junior.guru'
+
+
+def is_fetch_job(job):
+    return job.get('workflows', {}).get('job_name', '') == 'fetch'
+
+
+def is_nightly_job(job):
+    return job.get('workflows', {}).get('workflow_name', '') == 'nightly'
+
+
+def is_fetch_and_nightly(job):
+    return is_fetch_job(job) and is_nightly_job(job)
+
+
+def fetch_url(build_num):
+    try:
+        res = get(f'{PROJECT_URL}/{build_num}/artifacts')
+        artifacts = list(filter(lambda artifact: artifact.get('path', '') == 'backup.tar.gz', res.json())) 
+        # only one artifact should be there
+        artifact = artifacts[0] if len(artifacts) > 0 else {}
+
+        return artifact.get('url', None)
+    except Exception as e:
+        # something has failed, but no big deal
+        print (e);
+        pass
+
+
+def download_data(path, url):
+    with open(path, 'wb') as f:
+        for chunk in get(url, stream = True).iter_content(chunk_size = 1024*1024*3):
+            f.write(chunk)
+
+
+def fetch_artifacts(offset, loop):
+    os.makedirs(DOWNLOAD_DIR, exist_ok = True)
```

Comment:
Nemusíš opravovat, je to tu na víc místech, ale nevadí mi to. Spíš jen FYI, konvence je psát keyword arguments bez mezer kolem `=`, tedy v tomto případě:

```python
os.makedirs(DOWNLOAD_DIR, exist_ok=True)
```

Na tomto projektu žádná konvence není a nechci tu vůbec komentovat coding style, spíš jsem si jen všiml, že to celkem konzistentně píšeš s těmi mezerami, tak jen upozorňuji, že zvykem je psát to bez nich. Spíš kvůli tomu, abys na to nenarazil jinde.